common/src/main/scala/com/aliyun/emr/rss/common/protocol/message/ControlMessages.scala

```diff
@@ -364,10 +364,10 @@ sealed trait Message extends Serializable{
       case DestroyResponse(status, failedMasters, failedSlaves) =>
         val builder = TransportMessages.PbDestroyResponse.newBuilder()
           .setStatus(status.getValue)
-        if (failedMasters != null) {
+        if (!failedMasters.isEmpty) {
```

Comment:
There is no need to check whether the list has elements or not. A list called "addall" action with an empty list is safe.

Reply:
Done